From cf5eb66b34df4d1478940125099fb320832edf97 Mon Sep 17 00:00:00 2001
Message-Id: <cf5eb66b34df4d1478940125099fb320832edf97.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Shobhit Kumar <shobhit.kumar@intel.com>
Date: Mon, 10 Nov 2014 19:30:02 +0530
Subject: [PATCH 40/75] MUST_REBASE [VPG]: adf/intel: Send HPD event only for
 HDMI interfaces

By default the intel-adf framework will send HPD even for DSI, which is
not needed

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

For: GMINL-3547
Change-Id: I3c5c03f4c7e83cf703998ff382fb8284e7a5ccaf
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/video/adf/intel/intel_adf_interface.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/video/adf/intel/intel_adf_interface.c b/drivers/video/adf/intel/intel_adf_interface.c
index 86873e1..2236c00 100644
--- a/drivers/video/adf/intel/intel_adf_interface.c
+++ b/drivers/video/adf/intel/intel_adf_interface.c
@@ -252,7 +252,9 @@ static int set_preferred_mode(struct intel_adf_interface *intf)
 	if (err)
 		goto out_err0;
 
-	adf_hotplug_notify_connected(&intf->base, modelist, n_modes);
+	/* Only send HPD event in case of HDMI */
+	if (pipe->type == INTEL_PIPE_HDMI)
+		adf_hotplug_notify_connected(&intf->base, modelist, n_modes);
 
 	return 0;
 out_err0:
-- 
1.7.9.5

